src/forms/formHelpers.js

```diff
@@ -158,11 +159,12 @@ const FormPartsContainer = function(props) {
         </Header>
         { !props.clarifier
           ? null
-          : <Header
+          : 
```

Comment:
I'd prefer having an opening parenthesis here, so the `Header` under it is indented, and then a closing parenthesis on its own line after line 167. I think the indenting looks better that way. Anyone else have a preference?

Reply:
I'll give my thoughts on other stuff later, but I think the closing paren would be on a new newline after line 165.